as/json.js

```diff
@@ -91,6 +91,7 @@ TChannelJSON.prototype.send = function send(
 ) {
 
     var self = this;
+    req.openSpan = req.channel.startSpan(head);
```

Comment:
nit: if you pass `req` instead, the method can store the span on `req` instead of returning